tests/functional/test_adcm_check_plugin.py

```diff
@@ -20,10 +19,13 @@ def test_field_validation(sdk_client_fs: ADCMClient, missed_field):
     cluster = bundle.cluster_create(utils.random_string())
     task = cluster.action_run(name='adcm_check')
     task.wait()
-    assert task.status == 'failed', "Current job status {}. Expected: failed".format(task.status)
-    job = task.job()
-    logs = job.log_list()
-    assert len(logs) == 2, "Logs count not equal 2, current log count {}".format(len(logs))
+    with allure.step('Check if job status is failed'):
+        assert task.status == 'failed', \
+            "Current job status {}. Expected: failed".format(task.status)
```

Comment:
Hereinafter
Reason is too general. It would be better if specify name of action that failed

Reply:
done

Reply:
[ADCM-1557](https://arenadata.atlassian.net/browse/ADCM-1557)

Reply:
[ADCM-1569](https://arenadata.atlassian.net/browse/ADCM-1569)